wallet/src/account/mod.rs

```diff
@@ -2189,67 +2366,118 @@ fn group_preselected_inputs(
                     | TxOutput::CreateDelegationId(_, _)
                     | TxOutput::DelegateStaking(_, _)
                     | TxOutput::IssueFungibleToken(_)
-                    | TxOutput::DataDeposit(_) => {
+                    | TxOutput::DataDeposit(_)
+                    | TxOutput::AnyoneCanTake(_) => {
```

Comment:
Would CreateOrder be a better name than AnyoneCanTake?

Reply:
Originally it was CrateOrder but after some discussions with Sam it was renamed to AnyonCanTake

Reply:
>> Would CreateOrder be a better name than AnyoneCanTake?

> Originally it was CrateOrder but after some discussions with Sam it was renamed to AnyonCanTake

Maybe we should reconsider the name? `AnyoneCanTake` is way too cryptic IMO.